smrf/tests/output/test_netcdf.py

```diff
@@ -10,14 +11,11 @@
 
 class TestOutputNetCDF(SMRFTestCase):
 
-    @classmethod
-    def setUpClass(cls):
+    def setUp(self):
         super().setUpClass()
-        cls.smrf = SMRF(cls.config_file)
+        self.smrf = SMRF(self.config_file)
```

Comment:
Why did you need to switch to an instance here?

Reply:
It gets initialized with the 64-bit test first and doesn't get overwritten. So a new instance was needed to set all the config